card/db/migrate_core_cards/data/more_bootstrap.json

```diff
@@ -0,0 +1,51 @@
+      },
+      {
+        "name": "simple skin",
+        "type": "Skin",
+        "content": "[[style: jquery-ui-smoothness]]\n[[style: functional]]\n[[style: standard]]\n[[style: right sidebar]]\n[[style: common]]\n[[style: traditional]]\n[[style: glyphicons]]\n[[style: bootstrap compatible]]"
+      },
+      {
+        "name": "simple bootstrap skin",
+        "type": "Skin",
+        "content": "[[style: jquery-ui-smoothness]]\n[[style: functional]]\n[[style: standard]]\n[[style: right sidebar]]\n[[style: common]]\n[[style: bootstrap]]\n[[style: bootstrap theme]]\n[[style: bootstrap cards]]"
+      },
+      {
+        "name": "Default Layout",
+        "type": "Layout",
+        "content": "<!DOCTYPE HTML>\r\n<html>\r\n  <head>\r\n    {{*head}}\r\n  </head>\r\n  \r\n  <body>\r\n    <div id=\"menu\">\r\n      {{*page header}}\r\n    </div>\r\n    \r\n    <div id=\"primary\">\r\n      {{_main}}\r\n    </div>\r\n    \r\n    <div id=\"secondary\">\r\n      {{*sidebar}}\r\n    </div>\r\n  </body>\r\n</html>"
+      },
+      {
+        "name": "*all+*script",
+        "type": "Pointer",
+        "content": "[[script: jquery]]\n[[script: tinymce]]\n[[script: slot]]\n[[script: card menu]]\n[[script: jquery helper]]\n[[script: bootstrap]]\n[[script: ace]]"
+      },
+      {
+        "name": "*page header",
+        "type": "HTML",
+        "content": "<div class=\"container-fluid\">\r\n  <div class=\"navbar-header\">\r\n    <button type=\"button\" class=\"navbar-toggle collapsed\" data-toggle=\"collapse\" data-target=\"#navbar-collapse-1\">\r\n      <span class=\"sr-only\">Toggle navigation</span>\r\n        <span class=\"icon-bar\"></span>\r\n        <span class=\"icon-bar\"></span>\r\n        <span class=\"icon-bar\"></span>\r\n      </button>\r\n  </div>\r\n  <div class=\"collapse navbar-collapse\" id=\"navbar-collapse-1\">\r\n    <ul class=\"nav navbar-nav navbar-left\">\r\n      <li>[[/ | Home]]</li>\r\n      <li>[[/recent | Recent]]</li>\r\n    </ul>   \r\n    {{*navbox|navbar_left}}\r\n    \r\n    {{*account links|list_navbar_right}}\r\n  </div>\r\n</div>"
+      },
+      {
+        "name": "style: bootstrap compatible",
+        "type": "SCSS",
+        "content": "#menu {\r\n  height: 30px;\r\n  div.container-fluid{\r\n    display: block;\r\n  }\r\n  div.navbar-header {\r\n    display: none;\r\n  }\r\n  ul {\r\n    font-size: 14px;\r\n    margin-top: 5px;\r\n  }\r\n  li {\r\n    display: inline;\r\n  }\r\n  .navbar-left {\r\n    float: left;\r\n  }\r\n  .navbar-right {\r\n    float: right;\r\n    margin-right: 15px;\r\n  }\r\n  .navbar-form {\r\n    width: 30%;\r\n    margin-top: 5px;\r\n    margin-left: 10px;\r\n    input {\r\n      width: 100%;\r\n    }\r\n  }\r\n}\r\n\r\n#primary {\r\n  clear: both;\r\n}"
+      },
+      {
+        "name": "Old Layout",
+        "type": "Layout",
```

Comment:
For this one I was thinking to actually take their default layout and rename it (so that they still have whatever they've changed).

I'm pretty nervous that we're going to break pretty much any site that has done a lot of customizations.